eqxvision/models/classification/vgg16.py

```diff
@@ -0,0 +1,60 @@
+import jax
+import jax.numpy as jnp
+from equinox import Module
+from equinox.nn import MaxPool2D, Linear, Conv2d
+
+
+def preprocess_input(image, mean=[103.939, 116.779, 123.68]):
```

Comment:
This is usually taken care at the data loading side of things as people tend to apply different transformations as per their needs.

Reply:
If one uses the pre-trained weights and wants to use the model directly, then one would need this specific transformation.